app/components/App.js

```diff
@@ -2,7 +2,22 @@ import React from 'react';
 
```

Comment:
Please use 2-spaced indentation instead of 4. See [Google JS style guide](https://google.github.io/styleguide/jsguide.html#formatting-block-indentation)